00ecb4f54feff7c5c2e946866f0abbfcb629aea3
2026-08-21 10:08:58 +0200
Nj├Ñl Solland
license: name Nj├Ñl Gaute Solland as copyright holder

diff --git a/LICENSE b/LICENSE
index 261eeb9..51e3dd7 100644
--- a/LICENSE
+++ b/LICENSE
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2026 Nj├Ñl Gaute Solland
 
    Licensed under the Apache License, Version 2.0 (the "License");
    you may not use this file except in compliance with the License.
diff --git a/NOTICE b/NOTICE
index 08362e5..22ea77a 100644
--- a/NOTICE
+++ b/NOTICE
@@ -1,5 +1,5 @@
 PEACE Protocol
-Copyright 2026 PEACE Protocol contributors
+Copyright 2026 Nj├Ñl Gaute Solland
 
 This repository contains the public PEACE Protocol specification, schemas, conformance vectors and related reference material.
 
